plugins/fleet.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: fleet
+spec:
+  version: "v0.1.3"
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/mhausenblas/kcf/releases/download/v0.1.3/fleet_linux_amd64.tar.gz
+    sha256: "4b70b89fd78d3dc674ff5db918f0047dca816c1622dbbf81f957b1a1419cac39"
+    files:
+    - from: "./fleet"
+      to: "."
+    bin: "fleet"
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/mhausenblas/kcf/releases/download/v0.1.3/fleet_darwin_amd64.tar.gz
+    sha256: "6b1c2404a0a5a734f6313e765930b92877109ee396d290291d66d2139a74a18b"
+    files:
+    - from: "./fleet"
```

Comment:
Can you also include a LICENSE file in your bundle and extract that with your distribution, too?